guides/hack/11-built-in-types/22-arrays-examples/legacy-array-fruits.php

```diff
@@ -1,17 +0,0 @@
-<?hh // strict
-
-namespace Hack\UserDocumentation\Types\Arrays\Examples\LegAryFruits;
-
-<<__EntryPoint>>
-function main(): void {
-  $fruits = array(); // create an empty array
-  \var_dump($fruits);
-
-  $fruits['oranges'] = 22; // create element 4 with value "black"
-  \var_dump($fruits);
-
-  $fruits['oranges'] = 18; // replace element 4’s value with "red"
-  $fruits['apples'] = 15; // create element 8 with value "white"
-  $fruits['pears'] = 12; // create element -3 with value "blue"
```

Comment:
Might be worthwhile checking that all test files are used, as part of our build.

(I'm also puzzled why this says 'element -3' here, but whatever.)

Reply:
Yeah, I deleted this test as... it makes no sense and it's unreferenced.